docs/google-cloud.md

```diff
@@ -34,107 +44,121 @@ gcloud auth configure-docker
+```Creating cluster ... in ... Cluster is being health-checked...``` 
+
+You'll get a message when cluster creation has completed.
 
 While you wait, here's how the infrastructure works:
-1. The CTF challenges will run inside "nsjail" (a security sandbox).
-1. The contents of the nsjail sandbox will be configured in a docker container.
-1. The container will be deployed using Kubernetes in a group of VMs.
+1. The CTF challenges run inside "nsjail" (a security sandbox).
+1. The contents of the nsjail sandbox are configured in a docker container.
+1. The container is deployed using Kubernetes in a group of VMs.
 1. If the VMs consume too much CPU, Kubernetes automatically deploys more VMs.
-1. If the VMs consume too little CPU, Kubernetess will shut down some VMs.
-1. Some very expensive challenges will probably be allocated in the same VMs as less busy challenges.
-
-This all ensures the availability of the challenges, and saves some computing resources.
+1. If the VMs consume too little CPU, Kubernetes will shut down some VMs.
+1. Some very expensive challenges can be allocated to the same VMs as less busy challenges.
 
-Anyway, hopefully by now the cluster is already created and we can continue with the walkthrough.
+The above steps ensure the availability of the challenges, while using computing resources in a resourceful manner.
 
-If you are curious and have some more time, take a look at this [kCTF introduction](introduction.md), which includes a quick 8 minutes summary of what is kCTF and how it interacts with Kubernetes.
+By now, your cluster should be ready and we can continue with the walkthrough.
 
-# Step 2 - Create a challenge
-Now that we have a cluster setup, we need to create a challenge.
+Note: If you are curious and have some spare time, take a look at the [kCTF introduction](https://github.com/google/kctf/blob/master/docs/introduction.md), which includes a quick 8 minute summary of what kCTF is and how it interacts with Kubernetes.
 
-In this walkthrough, you'll learn how to create a challenge called "demo-challenge", build the docker image, deploy it to the cluster, and expose it to the internet.
+# Step 2 – Create a challenge
+Now that you have set up a cluster, you can create a challenge.
 
-You need the cluster to be created before you continue, otherwise the following commands won't work.
+In this step, you'll learn how to:
+* Create a challenge called "demo-challenge"
+* Build a Docker image
+* Deploy the Docker image to the cluster
+* Expose the Docker image to the internet
 
-Click next to continue if your cluster is already created.
+Note: The cluster must be created before you continue, otherwise the following commands won't work. To create a cluster, see the preceding steps. Continue with the next steps if you already created a cluster.
 
-## First, call create_challenge.sh to copy the skeleton
-Run the following command to create a challenge called "demo-challenge"
+## Call create_challenge.sh to copy the skeleton
```

Comment:
hmm seems like we changed the name of the command it's not called create_challenge.sh anymore

Reply:
kctf-chal-create?

Reply:
changed to kctf-chal-create